plugins/inputs/procstat/README.md

```diff
@@ -42,6 +42,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## CGroup name or path, supports globs
   # cgroup = "systemd/system.slice/nginx.service"
 
+  # Supervisor_unit name, specify the service name of the hypervisorctl management to be executed: webserver, proxy
+  # pid_finder = "pgrep" <- When you use it, this configuration may be necessary
+  # supervisor_unit = "webserver, proxy"
+
```

Comment:
This change needs to be done in `sample.conf` as this file is automatically embedded into the readme via `make docs`.

Furhermore, please use a `[]string` type to allow the easier `supervisor_units = ["webserver", "proxy"]` configuration form.